recipes/lsstdesc_weaklensingdeblending/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "lsstdesc.weaklensingdeblending" %}
+{% set version = "0.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/LSSTDESC/WeakLensingDeblending/archive/v{{ version }}.tar.gz
+  sha256: 2741773f3274ec6216667bc7d2e0e08c9e3ec279ef1a7c8bb6bffec8323ba68a
+
+build:
+  number: 0
+  skip: True  # [win]
```

Comment:
What's the reason for this not being `noarch: python`?  Missing dependencies? Then that's no problem and we can ignore the failing Windows CI on merge.

Reply:
Woops you are right!

Reply:
done!